pico/maven-plugin/src/main/java/io/helidon/pico/maven/plugin/ApplicationCreatorMojo.java

```diff
@@ -35,13 +35,13 @@
 public class ApplicationCreatorMojo extends AbstractApplicationCreatorMojo {
 
     /**
-     * The classname to use for the Pico {@link io.helidon.pico.Application} class.
+     * The classname to use for the Pico {@link io.helidon.pico.api.Application} class.
      * If not found the classname will be inferred.
      */
     @Parameter(property = PicoServicesConfig.NAME + ".application.class.name", readonly = true
                // note: the default value handling doesn't work here for "$$"!!
-//               defaultValue = DefaultApplicationCreator.APPLICATION_NAME
-    )
+               //               defaultValue = DefaultApplicationCreator.APPLICATION_NAME
```

Comment:
this reformatting does not look good, was that intentional?

Reply:
no, this was something I guess IDEA did also.

Reply:
next push

Reply:
pushed